src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -388,26 +388,17 @@ const renderPricing = (
   }
 }
 
-const renderEstimate = (estimatedPrice, user, mediator, size) => {
-  const justifyContent = size === "xs" ? "flex-start" : "flex-end"
+const renderEstimate = (estimatedPrice, user, mediator) => {
+  let body: JSX.Element
   if (user) {
-    return (
-      <Flex justifyContent={justifyContent}>
-        {estimatedPrice && (
-          <>
-            <Sans size="2">{estimatedPrice}</Sans>
-          </>
-        )}
-        {!estimatedPrice && (
-          <>
-            <Sans size="2">Estimate not available</Sans>
-          </>
-        )}
-      </Flex>
-    )
+    if (estimatedPrice) {
+      body = <Sans size="2">{estimatedPrice}</Sans>
+    } else {
+      body = <Sans size="2">Estimate not available</Sans>
+    }
   } else {
-    return (
-      <Link
+    body = (
+      <Clickable
```

Comment:
This is what was failing with eslint. `<Link` elements equate to anchors, which are intended to navigate to a new URL. The accessible recommendation is to use buttons instead of anchors if they have `onClick` handlers. Palette now has a `Clickable` component that renders a `<button` instead, but we can easily style it to look like an anchor.